azure-pipelines.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       # run doctests in the documentation
       # TODO: use following line for doctest ?
       pytest docs/*rst || echo "Ignoring exit status"
-      pytest --import-mode=importlib --cov=rlberry --cov-report xml rlberry
+      pytest --import-mode=importlib --cov=rlberry --cov-report xml rlberry --ignore=rlberry/tests/test_no_extra_install.py
```

Comment:
Maybe do not ignore it here and then it will figure in the coverage report.

Reply:
> Maybe do not ignore it here and then it will figure in the coverage report.

I had ignored it because it was duplicate testing of "base" agents and envs.
Following your message, to add it to the code coverage, I reorganized the agent tests into "base" and "extra", which removes the duplicates and allows you to test specifically the ones you want (the code coverage checks everything, and the CI "base" check only the "base" agents and envs).